msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/SignInErrors.kt

```diff
@@ -39,6 +39,8 @@ open class SignInError(
     fun isUserNotFound(): Boolean = this.errorType == ErrorTypes.USER_NOT_FOUND
 
     fun isInvalidCredentials(): Boolean = this.errorType == SignInErrorTypes.INVALID_CREDENTIALS
+
+    fun isInvalidUsername(): Boolean = this.errorType == ErrorTypes.INVALID_USERNAME
```

Comment:
Do we need a related sample app update PR?

Reply:
No, update to sample app repo is required.